include/oneapi/tbb/detail/_flow_graph_join_impl.h

```diff
@@ -1456,36 +1457,21 @@
     };  // join_node_base
 
     // join base class type generator
-    template<int N, template<class> class PT, typename OutputTuple, typename JP>
+    template<template<class> class PT, typename OutputTuple, typename JP>
```

Comment:
To be consistent with similar changes:
```suggestion
    template<template<class> class PortType, typename OutputTuple, typename JoinPolicy>
```